doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -451,3 +454,155 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
+     - Virtio console type device for data input and output.
+
+   * - "virtio-hyper_dmabuf"
+     - Virtio device that allows sharing data buffers between VMs using a dmabuf-like interface
+
+   * - "virtio-hdcp"
+     - Virtio High-bandwidth Digital Content Protection (HDCP) type device. HDCP
+       is technology intended to protect unauthorized duplication of high
+       definition (HD) video and audio as it travels across connections.
+
+   * - "virtio-heci"
+     - Virtio Host Embedded Controller Interface, parameters should be appended with the format
+       <bus>:<device>:<function>,d<0~8>.
+
+   * - "virtio-i2c"
+     - Virtio i2c type device,parameters with format like
+       <bus>[:<client_addr>[@<node>]][:<client_addr>[@<node>]][,<bus>[:<client_addr>[@<node>]][:<client_addr>][@<node>]]
+       should be appended. <bus> specifies the bus number for the native I2C adapter, e.g.2 means /dev/i2c-2.
+       <client_addr> specifies the address for the native client devices such as 1C, 2F...
+       "@" specifies the prefix for the ACPI node.
+       <node> specifies the ACPI node name supported in the current code. Currently,only cam1,
+       cam2, and hdac are supported for APL platform. These nodes are platform-specific.
+
+   * - "virtio-gpio"
+     - Virtio GPIO type device, parameters format is like
+       virtio-gpio,<@controller_name{offset|name[=mapping_name]:offset|name[=mapping_name]:...}@controller_name{...}...]>
+       "controller_name", input ls /sys/bus/gpio/devices to check native gpio controller information. Usually,
+       the devices represent the controller_name, you can use it as controller_name directly. You can also
+       input cat /sys/bus/gpio/device/XXX/dev to get device id that can be used to match /dev/XXX, then use
+       XXX as the controller_name. On APL platform and Intel NUC platforms, the controller_name are gpiochip0,
+       gpiochip1, gpiochip2.gpiochip3.
+       "offset|name",you can use gpio offset or its name to locate one native gpio within the gpio controller.
+       "mapping_name",this is optional, if you want to use a customized name for a FE gpio, you can set a
+       new name for a FE virtual gpio.
+
+   * - "virtio-rnd"
+     - Virtio random generater type device, with string "kernal=on" to initialize virtio random generater state.
+
+   * - "virtio-rpmb"
+     - Virtio Replay Protected Memory Block (RPMB) type device, with "physical_rpmb" to specify RPMB in physical mode,
+       otherwise RPMB is in simulated mode.
+
+   * - "virtio-audio"
+     - Virtio audio type device
+
+   * - "virtio-net"
+     - Virtio network type device, parameter should be appended with the format
+       virtio-net,<tap_name|macvtap_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]
+       mac address is optional,"tap_name" needs to include the keyword tap.
+       vhost specifies vhost backend, otherwise use VBSU backend.
+
+   * - "passthru"
+     - Passthrough device, parameters below could be appended.
```

Comment:
You describe the optional flags below but you should also describe the BDF info that needs to be provided, e.g. `-s 2,passthru,0/17/0`